src/clients/api/mutations/vote/useCastVote.ts

```diff
@@ -29,6 +31,18 @@ const useCastVote = (options?: Options) => {
       onSuccess: (...onSuccessParams) => {
         const { proposalId, voteType } = onSuccessParams[1];
 
+        let voteTypeWord: AnalyticEventProps<'Vote casted'>['voteType'] = 'abstain';
+        if (voteType === 0) {
```

Comment:
Could also be defined and used as an enum

Reply:
Good point, that would be cleaner too. I'll make the change